crypto/ec/curve25519.c

```diff
@@ -670,60 +670,93 @@ static void fe_invert(fe out, const fe z) {
   fe t3;
   int i;
 
+  /*
+   * Compute z ** -1 = z ** (2 ** 255 - 19 - 2) with the exponent as
+   * 2 ** 255 - 21 = (2 ** 5) * (2 ** 250 - 1) + 11.
+   */
+
+  /* t0 = z ** 2 */
   fe_sq(t0, z);
-  for (i = 1; i < 1; ++i) {
-    fe_sq(t0, t0);
-  }
+
+  /* t1 = t0 ** 3 = z ** 8 */
   fe_sq(t1, t0);
   for (i = 1; i < 2; ++i) {
```

Comment:
Removed; like the dead loops, this was just for unifying the representation. I think we don't need it now that we have comments.